### ISSUE TITLE
unnecessary warning about nested-ness from mxCompare

### ISSUE THREAD

**Issue by tbates** [OpenMx/OpenMx#20]
mxCompare is the preferred way to see an AIC comparison of models. But mxCompare now warns users to use AIC comparison... Can we remove this warning? Or establish who wants to be warned about this?

```splus
mxCompare(m2, m1) # ADE is better
  base comparison ep minus2LL   df      AIC    diffLL diffdf  p
1  ADE       <NA>  4 9658.230 3851 1956.230        NA     NA NA
2  ADE        ACE  4 9659.215 3851 1957.215 0.9849158      0 NA
Warning message:
In collectStatistics(nextBaseSummary, nextCompareSummary) :
  Model 'ADE' has the same degrees of freedom as ACE which means the models are not nested and should not be compared with the likelihood ratio test. Compare these models using the information criteria statistics

```

**Comment by jpritikin** [comment]
On Tue, Apr 04, 2017 at 01:12:49PM -0700, Tim Bates wrote:
>    mxCompare is the preferred way to see an AIC comparison of models. But
>    mxCompare now warns users to use AIC comparison... Can we remove this
>    warning? Or establish who wants to be warned about this?
> 
> mxCompare(m2, m1) # ADE is better
>   base comparison ep minus2LL   df      AIC    diffLL diffdf  p
> 1  ADE       <NA>  4 9658.230 3851 1956.230        NA     NA NA
> 2  ADE        ACE  4 9659.215 3851 1957.215 0.9849158      0 NA
> Warning message:
> In collectStatistics(nextBaseSummary, nextCompareSummary) :
>   Model 'ADE' has the same degrees of freedom as ACE which means the models are
> not nested and should not be compared with the likelihood ratio test. Compare th
> ese models using the information criteria statistics

Instead of a warning, how about if we set diffLL to NA when df are 
equal?

-- 
Joshua N. Pritikin, Ph.D.
Virginia Institute for Psychiatric and Behavioral Genetics
Virginia Commonwealth University
PO Box 980126
800 E Leigh St, Biotech One, Suite 1-133
Richmond, VA 23219
http://exuberant-island.surge.sh

**Comment by tbates** [comment]
Just setting the  p-value to NA (as it happening) is plenty of warning to people, without any other information.

I would just not print the warning at all. The only candidates for being helped by it are new-users, who are instead terrified by it, unnecessarily.

Definitely don't put fake numbers in for Δ-2ll: Much to easy to misinterpret as factual.

The difference often is far from  zero (which is what drives the AICs apart). It's only the p-value which we wish to direct people away from, and this is set to NA (used to be a very small actual value  like 5e-17 or so)

**Comment by jpritikin** [comment]
Fixed in v2.7.9-72-g1ff36c605
